src/shared_utilities/drivers.jl

```diff
@@ -289,6 +289,43 @@ function turbulent_fluxes!(
 end
 
 
+function coupler_compute_turbulent_fluxes!(
```

Comment:
This is basically a copy of `turbulent_fluxes!` with `atmos::PrescribedAtmos`. We also have `turbulent_fluxes!` with `atmos::CoupledAtmos` which does nothing (as the value has been precomputed by coupler. These functions are called in the `update_boundary_fluxes` function of the land model.

I liked giving this another name, though, because it is called in a different part of the code and used in a different way, sort of.